karrot/places/models.py

```diff
@@ -13,6 +14,32 @@ class PlaceStatus(models.TextChoices):
     ARCHIVED = 'archived'
 
 
+class PlaceTypeStatus(models.TextChoices):
+    ACTIVE = 'active'
+    ARCHIVED = 'archived'
+
+
+class PlaceType(BaseModel, UpdatedAtMixin):
+    DEFAULT_STATUS = PlaceTypeStatus.ACTIVE.value
+
+    group = models.ForeignKey('groups.Group', on_delete=models.CASCADE, related_name='place_types')
+    name = models.CharField(max_length=80)
+    name_is_translatable = models.BooleanField(default=True)
+    icon = models.CharField(max_length=32)
```

Comment:
Maybe 32 characters can be too short for some icons? Just had a look, there's `fas fa-american-sign-language-interpreting` in https://quasarframework.github.io/quasar-ui-qiconpicker/icons, so we might just push max_length to 100 to be safe.

(I guess that also affects `ActivityType`?)

Reply:
Yup, will switch to 100.

Reply:
Fixed in https://github.com/yunity/karrot-backend/pull/1159/commits/d3b2d39f70d704552284c77db9429dfc83e3effa